wagtail_airtable/mixins.py

```diff
@@ -343,6 +345,17 @@ def save(self, *args, **kwargs):
                     # Used in the `after_edit_page` hook. If it exists, an error message will be displayed.
                     self._airtable_update_error = message
 
+    def save(self, *args, **kwargs):
+        """
+        If there's an existing airtable record id, update the row.
+        Otherwise attempt to create a new record.
+        """
+        saved_model = super().save(*args, **kwargs) # Save to database first so we get pk, in case it's used for uniqueness
+
+        if settings.get("WAGTAIL_AIRTABLE_SAVE_SYNC", True):
```

Comment:
This works - I was actually thinking this would point to a function, which by default would just call `self.save_to_airtable`. Works either way, though...